src/frontend/parser.mly

```diff
@@ -33,6 +43,8 @@ let reducearray (sbt, l) =
 %token PRINT REJECT
 %token TRUNCATE
 %token EOF
+
+(* Unreachable tokens are useful for differentiating states *)
```

Comment:
Interesting, is there some online resource we could link to explaining why that is? I realize you probably weren't the person to add this but if you know of one and can add it that would be swell.

Reply:
I was actually the person to add this in #457, and I don't know of an online resource. I explained it a bit in the PR but I could definitely add a paragraph comment here.